### PR TITLE
Fix issue with including libraries in resources directory

### PR DESCRIPTION
**Pull request by warunalakshitha**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/20521

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [x] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/PathDependencyTestCase.java [hunk 1]
```diff
@@ -39,14 +39,15 @@
...
-import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
-import static java.util.concurrent.TimeUnit.SECONDS;
 import static org.awaitility.Awaitility.given;
 import static org.ballerinalang.test.packaging.PackerinaTestUtils.deleteFiles;
 import static org.wso2.ballerinalang.compiler.util.ProjectDirConstants.BLANG_COMPILED_JAR_EXT;
 import static org.wso2.ballerinalang.compiler.util.ProjectDirConstants.BLANG_COMPILED_PKG_BINARY_EXT;
 import static org.wso2.ballerinalang.compiler.util.ProjectDirConstants.BLANG_SOURCE_EXT;
 
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+import static java.util.concurrent.TimeUnit.SECONDS;
```
Comment (hemikak) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/PathDependencyTestCase.java]: Are these imports used ?

File: tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+org-name = "toml"
+version = "0.7.2"
+
+[platform]
+target = "java8"
+
+  [[platform.libraries]]
+  artifactId = "toml4j"
+  version = "0.7.2"
+  path = "./src/toml4j/resources/libs/toml4j-0.7.2.jar"
```
Comment (hemikak) [inline tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/Ballerina.toml]: Where have we committed these JARs in the PR ?

File: tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/Ballerina.toml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  groupId = "com.moandjiezana.toml"
+  module = "toml4j"
+
+  [[platform.libraries]]
+  artifactId = "gson"
+  version = "2.8.1"
+  path = "./src/toml4j/resources/libs/gson-2.8.1.jar"
```
Comment (hemikak) [inline tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/Ballerina.toml]: Where have we committed these JARs in the PR ?

File: tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/Ballerina.toml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  groupId = "com.google.code.gson"
+  module = "toml4j"
```
Comment (hemikak) [inline tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/Ballerina.toml]: Add a new line.

File: tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/src/toml4j/funcs.bal [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    name: "read",
+    class: "com.moandjiezana.toml.Toml",
+    paramTypes: ["java.lang.String"]
+} external;
+
+
+public function getString(handle instance, handle key) returns handle = @java:Method {
+  name: "getString",
+  class: "com.moandjiezana.toml.Toml"
+} external;
```
Comment (hemikak) [inline tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/src/toml4j/funcs.bal]: Add a new line.

File: tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject2/Ballerina.toml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+[project]
+org-name = "foo"
+version = "2.0.0"
+
+[dependencies]
+"toml/toml4j" = { path = "../TestProject1/target/balo/toml4j-2019r3-java8-0.7.2.balo"}
```
Comment (hemikak) [inline tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject2/Ballerina.toml]: Add a new line.

File: tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/Ballerina.toml (hunk 3 above)
Comment (warunalakshitha) [reply tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/Ballerina.toml]: Seems missied due to git ignore. Added with latest commit

File: tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/Ballerina.toml (hunk 2 above)
Comment (warunalakshitha) [reply tests/jballerina-integration-test/src/test/resources/packaging/balopath/case9/TestProject1/Ballerina.toml]: Seems missed due to git ignore. Added with latest commit

**Pull request merged**